mkdocs-project-dir/docs/howto.md

```diff
@@ -9,6 +14,7 @@ Logging in is most straightforward if you are inside the UCL firewall. If you ar
 ### Linux / Unix / Mac OS X
 
 Use the terminal and type the below command to secure shell (ssh) into the machine you wish to access. Replace `<your_UCL_user_id>` with your central UCL username, and `<system_name>` with the name of the machine you want to log in to, eg. `legion`, `grace`, `aristotle`. 
+<!--- myriad, kathleen, thomas, michael, young  --->
```

Comment:
I mentioned in Slack, but this should just mention the UCL-internal machines and not MMM ones. Myriad, Kathleen.